.buildkite/test.sh

```diff
@@ -1,80 +1,67 @@
 #!/bin/bash
 # This script is used to build ddev/ddev using buildkite
 
+set -eu -o pipefail
+
 export PATH=$PATH:/home/linuxbrew/.linuxbrew/bin
 
-# GOTEST_SHORT=8 means drupal9
-export GOTEST_SHORT=8
+# GOTEST_SHORT=12 means drupal0
+export GOTEST_SHORT=12
 
 export DOCKER_SCAN_SUGGEST=false
 export DOCKER_SCOUT_SUGGEST=false
 
 # On macOS, we can have several different docker providers, allow testing all
 if [ "${OSTYPE%%[0-9]*}" = "darwin" ]; then
   function cleanup {
-#      docker context use default
+    orb stop &
+    killall com.docker.backend || true
+    colima stop || true
+    colima stop vz || true
+    limactl stop lima-vz || true
+    ~/.rd/bin/rdctl shutdown || true
+    docker context use default
+    # Leave orbstack running as the most likely to be reliable
+    orb start &
```

Comment:
`orb` is not installed on all runners:

```suggestion
    command -v orb &>/dev/null && orb start &
```